src/pivotal-ui/components/iconography/iconography.scss

```diff
@@ -156,4 +156,6 @@ There are a bunch of little cartoon people that we can use as avatar icons throu
   height: 100px;
   width: 100px;
   display: inline-block;
+  background-size: contain;
```

Comment:
It scales the background image to fit entirely within the bounding box, rather than cropping it.